### DISCUSSION TITLE
Contribution question: production website

### DISCUSSION THREAD

**Discussion by marcelmoreau** [Stefanye/CSSHell#17] [Q&A] [answered]
Awesome project idea. Thanks for doing this.

For contributions, when you say "All examples are must be copied from websites found in production.", does that mean the author provides a codepen/jsfiddle illustrating the issue, or does it need to be from a "real" website (e.g. abcxyzcorporation.com)?

**Comment by cat-a-flame** [comment, marked as answer]
The idea is that all the examples are real, often encountered issues, and not made up ones. I want to highlight the common mistakes people make when working with CSS and help them to be aware of these and fix those. Some made up code or fooling around is not really helpful I think :)
